Source/ConversationV1/Models/DialogNodeNextStep.swift

```diff
@@ -21,6 +21,7 @@ public struct DialogNodeNextStep {
 
     /// How the `next_step` reference is processed.
     public enum Behavior: String {
+        // swiftlint:disable identifier_name
```

Comment:
I think this disables the `identifier_name` rule until the end of the file or until the linter sees a matching enable comment. Might be better to use `:previous`, `:this`, or `:next` instead.

```swift
case to = "jump_to" // swiftlint:disable:this identifier_name
```

Although I suspect this violation will disappear when we generate the Discovery service and it becomes:

```swift
case jumpTo = "jump_to"
```

Reply:
Good catch.  I will fix.